samza-api/src/main/java/org/apache/samza/operators/windows/Window.java

```diff
@@ -66,13 +65,12 @@
  * <p> Use the {@link Windows} APIs to create various windows and the {@link org.apache.samza.operators.triggers.Triggers}
  * APIs to create triggers.
  *
- * @param <M> the type of the input {@link MessageEnvelope}
- * @param <K> the type of the key in the {@link MessageEnvelope} in this {@link org.apache.samza.operators.MessageStream}.
+ * @param <M> the type of the input message
+ * @param <K> the type of the key in the message in this {@link org.apache.samza.operators.MessageStream}.
```

Comment:
I would agree w/ this comment. However, I will leave this to @vjagadish1989 to make the final call, since he is working on a patch for window/triggers.